src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -2498,11 +2498,6 @@ static float Vectorized512Small(ref float xRef, ref float yRef, nuint remainder)
         private static float MinMaxCore<TMinMaxOperator>(ReadOnlySpan<float> x)
             where TMinMaxOperator : struct, IAggregationOperator
         {
-            if (x.IsEmpty)
-            {
-                ThrowHelper.ThrowArgument_SpansMustBeNonEmpty();
-            }
```

Comment:
Why move this into each of the call sites?

Reply:
It matches how `IndexOf*` was doing is empty checks.

Reply:
I'd rather go the other way and move those IndexOf* checks into the helper.  We want / expect the entry point methods to be inlined into the caller, and for Min/Max it's not worth bloating each call site with the extra argument validation for an error condition.  For IndexOf* where an empty input isn't an error condition, if we think it's going to be very common for an empty input, then we can just leave the disparity, or if we really want them to have the same shape, we could move those checks for IndexOf into the helper and dedup from the entrypoints.

Reply:
Fixed. There's still a few `Aggregate` functions that don't have their helpers in the main template, but that's because aggregates don't always throw.